subprojects/core/src/main/java/org/gradle/api/internal/project/DefaultProject.java

```diff
@@ -588,10 +588,6 @@ public RoleBasedConfigurationContainerInternal getConfigurations() {
     @Deprecated
     @Override
     public org.gradle.api.plugins.Convention getConvention() {
-        DeprecationLogger.deprecateMethod(Project.class, "getConvention()")
```

Comment:
@eskatos Does this make sense to you, or am I going too far here? All places that I have encountered were logged for the same line where we also reported deprecation of the  `Convention` type.

Reply:
It does make sense to me